std-bits/table/src/main/java/org/enso/table/write/ExcelWriter.java

```diff
@@ -510,15 +505,15 @@ private static void writeValueToCell(Cell cell, int j, Storage<?> storage, Workb
         case Double d -> cell.setCellValue(d);
         case Long l -> cell.setCellValue(l);
         case LocalDateTime ldt -> {
-          cell.setCellValue(ldt);
+          cell.setCellValue(ExcelUtils.toExcelDateTime(ldt));
           cell.setCellStyle(getDateTimeStyle(workbook, "yyyy-MM-dd HH:mm:ss"));
         }
```

Comment:
Unrelated to this change but I'm surprised we have this `LocalDateTime` branch but no `ZonedDateTime` one.

Is this code even reachable? I don't imagine under what circumstances we can store `LocalDateTime` in a Column.

1) in a `value_type=Date_Time` we get `DateTimeStorage` and it stores `ZonedDateTime`
2) in a mixed column, the values are passed through Enso interop which only has `EnsoDateTime` that gets converted on Java boundary to `ZonedDateTime` again.

`LocalDateTime` feels like a relic that we have from times from before we had proper date-time interop in the engine. Or am I missing something?

Reply:
Agree was surprised too - but harmless to leave it in the Excel writer.

Reply:
Does anything break if you remove it? If it's dead code it would be better to remove it to not have to maintain it.